docs/docs/usage/scaladoc/settings.md

```diff
@@ -134,3 +142,28 @@ Example JSON file:
 }
 ```
 
+##### -snippet-compiler
+
+Snippet compiler arguments provide a way to configure snippet checking.
+
+This setting accept list of arguments in format:
+args := arg{,arg}
+arg := [path=]flag
+where path is a prefix of source paths to members to which argument should be set.
+
+If path is not present, argument will be used as default.
+
+Available flags:
+compile - Enables snippet checking.
+nocompile - Disables snippet checking.
+fail - Enables snippet checking, asserts that snippet doesn't compile.
```

Comment:
Could you please run this text through a spell/grammar checker?

Also, I suggest providing a concrete example of usage of this setting.

Reply:
Added